processors/src/main/java/io/opentelemetry/contrib/eventbridge/EventToSpanEventBridge.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.eventbridge;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.api.common.Value;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.exporter.internal.marshal.MarshalerWithSize;
+import io.opentelemetry.exporter.internal.otlp.AnyValueMarshaler;
+import io.opentelemetry.sdk.logs.LogRecordProcessor;
+import io.opentelemetry.sdk.logs.ReadWriteLogRecord;
+import io.opentelemetry.sdk.logs.data.LogRecordData;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A processor that records events (i.e. log records with an {@code event.name} attribute) as span
+ * events for the current span if:
+ *
+ * <ul>
+ *   <li>The log record has a valid span context
+ *   <li>{@link Span#current()} returns a span where {@link Span#isRecording()} is true
+ * </ul>
+ *
+ * <p>The event {@link LogRecordData} is converted to attributes on the span event as follows:
+ *
+ * <ul>
+ *   <li>{@code event.name} attribute is mapped to span event name
+ *   <li>{@link LogRecordData#getTimestampEpochNanos()} is mapped to span event timestamp
+ *   <li>{@link LogRecordData#getAttributes()} are mapped to span event attributes, excluding {@code
+ *       event.name}
+ *   <li>{@link LogRecordData#getObservedTimestampEpochNanos()} is mapped to span event attribute
+ *       with key {@code log.record.observed_timestamp}
+ *   <li>{@link LogRecordData#getSeverity()} is mapped to span event attribute with key {@code
+ *       log.record.severity_number}
```

Comment:
> Maybe you're just trying to maintain consistency with the protos?

Exactly... if / when the spec comes out and tell us how to translate between events and span events, we'll of course follow those instructions. But until then, the less opinionated we can be the better, and following the proto field naming is a way to avoid having an opinion. 

https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto#L161